src/app/server/Dnssd.cpp

```diff
@@ -501,15 +502,16 @@ void DnssdServer::StartServer(Dnssd::CommissioningMode mode)
     }
 }
 
-#if CHIP_ENABLE_ROTATING_DEVICE_ID && defined(CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID)
```

Comment:
This check is in two more places in this file. Would be good to remove it there for consistency.